recipes/eigency/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "eigency" %}
+{% set version = "2.0.0a1" %}
+{% set eigenversion = "3.2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ eigenversion }}.{{ version }}
```

Do not do this. This version is for the package version only.

If you want to indicate a build variant, i.e. that this version is built against eigen3.2 instead of eigen3.3 or something, then you should use [a custom build string](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string). BUT, you aren't building against more than one eigen version, so this is not needed.

```suggestion
  version: {{ version }}
```